spec/support/matchers/performance_matchers.rb

```diff
@@ -0,0 +1,28 @@
+# Generally used for performance tests, eg:
+# expect(time_seconds).to be_faster_than(:local => 1, :ci => 1.4) # typically CI is about 1.4 times slower than local
+#
+# Works well with performance helpers, eg:
+# require 'performance_helper'
+# ...
+# it 'finishes in a reasonable amount of time' do
+#   start_profiling
+#   do_stuff
+#   result = stop_profiling('profile_name')
+#   expect(total_time(result)).to be_faster_than(:local => 1, :ci => 1.4) # typically CI is about 1.4 times slower than local
+# end
+
+RSpec::Matchers.define :be_faster_than do |seconds_in_environments|
+  match do |actual|
+    if ENV.fetch("CI", false)
```

Comment:
I thought about doing that, but then I thought it could be confusing for someone reading `expect(total_time(result)).to be_faster_than(4)` to then find a pass on CI where it took 5s. Also, I'm not sure if the 1.4x rule holds for all types of code. Maybe if the DB is not involved it doesn't, for example. This way gives us more flexibility as well as being clearer.